### PR TITLE
add Review icon + Review handling in progressionPage

### PR DESCRIPTION
**Pull request by scalabw**
https://trello.com/c/KSe4Ltlb/2858-ajouter-%C3%A9toiles-de-review-dans-le-compte-de-lendpoint-utilis%C3%A9-par-la-page-activit%C3%A9

**Detailed purpose of the PR**
 
image

This PR add the handling of showing the review in the progression Page.

**Result and observation**

**Testing Strategy**

- [X] Unit testing

### CODE REVIEW

File: packages/@coorpacademy-nova-icons/icons.js [hunk 1]
```diff
@@ -245,7 +245,11 @@ const icons: Array<Icon> = [
   {filePath: path.resolve('./third-party/nova-solid.iconjar/icons/sd-card.svg')},
   {filePath: path.resolve('./third-party/nova-solid.iconjar/icons/folders.svg')},
   {filePath: path.resolve('./third-party/nova-solid.iconjar/icons/file-block-2.svg')},
-  {filePath: path.resolve('./third-party/nova-line.iconjar/icons/cookie.svg')}
+  {filePath: path.resolve('./third-party/nova-line.iconjar/icons/cookie.svg')},
+  // review
```
Comment (EmelineLeduc) [inline packages/@coorpacademy-nova-icons/icons.js:249]: Comment to delete

**Review by EmelineLeduc — CHANGES_REQUESTED** [review]

File: packages/@coorpacademy-nova-icons/icons.js (hunk 1 above)
Comment (scalabw) [reply packages/@coorpacademy-nova-icons/icons.js:249]: this is to separate the review icons from others you can see upper in the file that there is already some others to separate like podcast and so

Comment (scalabw) [reply packages/@coorpacademy-nova-icons/icons.js:249]: image

Comment (scalabw) [reply packages/@coorpacademy-nova-icons/icons.js:249]: you are right !

**Review by EmelineLeduc — APPROVED** [review]

**Review by esa-coorp — APPROVED** [review]

**Pull request merged**